src/app/legacy/psammead/psammead-locales/moment/ro.ts

```diff
@@ -0,0 +1,23 @@
+import moment from 'moment';
+import 'moment/locale/ro';
+
+moment.updateLocale('ro', {
+  relativeTime: {
+    future: 'peste %s',
+    past: 'acum %s',
+    s: 'câteva secunde',
+    ss: '%d secunde',
+    m: 'un minut',
+    mm: '%d minute',
+    h: 'o oră',
+    hh: '%d ore',
+    d: 'o zi',
+    dd: '%d zile',
+    M: 'o lună',
+    MM: '%d luni',
+    y: 'un an',
+    yy: '%d ani',
+  },
+});
+
+export default moment;
```

Comment:
Don't think this is required. Not sure if it has any affects exporting it, but the other locales don't.

Reply:
Just seeing the test failures, I think if you follow the format in https://github.com/bbc/simorgh/blob/07f050fb490f581dd2bc3d6c10745e9c0096a1b8/src/app/legacy/psammead/psammead-locales/moment/templates/update-locale/index.test.ts it should be ok.